public/app/core/components/ForgottenPassword/ForgottenPassword.tsx

```diff
@@ -40,29 +76,50 @@ export const ForgottenPassword: FC = () => {
       </div>
     );
   }
+
   return (
-    <Form onSubmit={sendEmail}>
-      {({ register, errors }) => (
-        <>
-          <Legend>Reset password</Legend>
-          <Field
-            label="User"
-            description="Enter your information to get a reset link sent to you"
-            invalid={!!errors.userOrEmail}
-            error={errors?.userOrEmail?.message}
-          >
-            <Input placeholder="Email or username" {...register('userOrEmail', { required: true })} />
-          </Field>
-          <HorizontalGroup>
-            <Button>Send reset email</Button>
-            <LinkButton fill="text" href={loginHref}>
-              Back to login
-            </LinkButton>
-          </HorizontalGroup>
+    <>
+      <Form onSubmit={sendEmail}>
+        {({ register, errors }) => (
+          <>
+            <Legend>Reset password</Legend>
+            <Field
+              label="User"
+              description="Enter your information to get a reset link sent to you"
+              invalid={!!errors.userOrEmail}
+              error={errors?.userOrEmail?.message}
+            >
+              <Input placeholder="Email or username" {...register('userOrEmail', { required: true })} />
```

Comment:
```suggestion
              <Input id={'userOrEmail'} placeholder="Email or username" {...register('userOrEmail', { required: true })} />
```
Our `Field` component handles the logic of connecting the label to the input inside of it, but for this to work, we need to provide the `id` attribute.

Reply:
Fixed!